exporter/splunkhecexporter/README.md

```diff
@@ -22,6 +22,8 @@ The following configuration options can also be configured:
 - `disable_compression` (default: false): Whether to disable gzip compression over HTTP.
 - `timeout` (default: 10s): HTTP timeout when sending data.
 - `insecure_skip_verify` (default: false): Whether to skip checking the certificate of the HEC endpoint when sending data over HTTPS.
+- `auth_header` (default: Authorization): Optional custom authentication header.
+- `auth_token` (default: Splunk \<hec_token\>) Optional custom authentication token. Replaces `token` value.
```

Comment:
It is not clear why we need this in addition to `token` setting. What's the interplay between these 3 settings?